templates/default.yaml

```diff
@@ -279,6 +287,17 @@ containerd:
 # 🟢 Builtin default: not set
 minimumLimaVersion: null
 
+# EXPERIMENTAL
+# Default settings can be imported from base templates. These will be merged in when the instance
+# is created, and the combined template is stored in the instance directory.
+# This setting ca be either a single string (URL), or a list of locators.
+# A locator is again either a string (URL), or an object with "url" and "digest" properties, e.g.
+# base: [{url: ./base.yaml, digest: decafbad}, …]
+# The "digest" property is currently unused.
```

Comment:
Should print an error when the property is set?

Reply:
I can do that, but then I could have left-out the property all together; I only added it to show you how it will work.

I can implement it once this PR is merged, but I didn't want to include it because this PR is already way too big.

Anyways, let me know if you want to add an error to validation in this PR.